.editorconfig

```
@@ -0,0 +1,61 @@
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = false
```

We should ensure EOL on last line is present.

```suggestion
insert_final_newline = true
```